pkg/kt/snmp.go

```diff
@@ -159,13 +159,14 @@ type EAPIConfig struct {
 }
 
 type MerakiConfig struct {
-	ApiKey            string   `yaml:"api_key"`         // Required.
-	Host              string   `yaml:"host"`            // Optional, defaults to api.meraki.com
-	MonitorUplinks    bool     `yaml:"monitor_uplinks"` // This will be the default if neither is set.
-	MonitorDevices    bool     `yaml:"monitor_devices"`
-	MonitorOrgChanges bool     `yaml:"monitor_org_changes"`
-	Orgs              []string `yaml:"organizations"` // Only monitor orgs in this list, if set.
-	Networks          []string `yaml:"networks"`      // Only monitor networks in this list, if set.
+	ApiKey                string   `yaml:"api_key"`         // Required.
+	Host                  string   `yaml:"host"`            // Optional, defaults to api.meraki.com
+	MonitorUplinks        bool     `yaml:"monitor_uplinks"` // This will be the default if neither is set.
+	MonitorDevices        bool     `yaml:"monitor_devices"`
+	MonitorOrgChanges     bool     `yaml:"monitor_org_changes"`
+	MonitorNetworkClients bool     `yaml:"monitor_clients"`
```

Comment:
These map to 3 separate API calls:

`monitor_uplinks` -> https://developer.cisco.com/meraki/api-v1/get-organization-uplinks-statuses/
`monitor_clients ` -> https://developer.cisco.com/meraki/api-v1/get-network-clients/ 
`monitor_devices` -> https://developer.cisco.com/meraki/api-v1/get-device-clients/ 

`monitor_uplinks` is probably always good to turn on. This gives the overall health of each meraki box and how it talks to the internet. Basically "is this site online?" and "how much overall traffic is it doing / how's the latency?"

`monitor_clients` and `monitor_devices` give basically the same info: "what devices (eg, Mile's iphone) are running in each site? / how much traffic is each device pulling?"

The difference is that `monitor_devices` pulls up each application for each device and breaks traffic out by this. The problem is that doing this call by application turns out to be super slow. So if you have a large number of devices, it will never return any data.  I would recommend that `monitor_clients` is always used unless this per application data is needed.

Reply:
As a use-case for `monitor_clients`, here's an example of a big top talker, Alex's phone using data: 

<img width="182" alt="image" src="https://github.com/kentik/ktranslate/assets/315460/0aa3ed5d-7f6c-4684-8df6-d9064448a48a">